src/api/app/controllers/webui/project_controller.rb

```diff
@@ -226,7 +226,7 @@ def restore
   end
 
   def edit
-    authorize @project, :update?
+    authorize @project, :edit?
```

Comment:
Why? Do you expect this this to be different?

BTW you can also skip the method argument and it'll be derived from the controller method name you call this in.

Reply:
I just created `:edit?` to be consistent with names, but it's true it's exactly the same as using `:update?`. I'll remove the one I created.